stable/cert-manager-setup/templates/v1upgrade.yaml

```diff
@@ -122,5 +125,35 @@ metadata:
     "helm.sh/hook": pre-install, post-install, post-upgrade
     "helm.sh/hook-weight": "-11"
     "helm.sh/hook-delete-policy": hook-succeeded,before-hook-creation
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "cert-manager-setup.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "cert-manager-setup.labels" . | indent 4 }}
+  annotations:
+    "helm.sh/hook": post-install,post-upgrade
+    "helm.sh/hook-weight": "-6"
+    "helm.sh/hook-delete-policy": hook-succeeded,before-hook-creation
+spec:
+  template:
+    metadata:
+      name: "wait-for-cert-manager-webhook"
+    spec:
+      serviceAccountName: default
+      restartPolicy: Never
+      containers:
+      - name: {{ .Chart.Name }}
```

Comment:
do we want to name these a bit more specifically so it's clearer what the job is doing?